Makefile

```diff
@@ -62,6 +63,33 @@ lint:
 release-major release-minor release-patch:
 	@$(XYZ) --increment $(@:release-%=%)
 
+.PHONY: bundle
+bundle:
+	mkdir dist || echo "Directory already exists"
+	cat \
+  node_modules/sanctuary-type-identifiers/index.js \
+  node_modules/sanctuary-type-classes/index.js \
+  node_modules/sanctuary-def/index.js \
+  index.js \
+  >dist/sanctuary.js
+	cat dist/sanctuary.js | $(UGLIFYJS) -mc > dist/sanctuary.min.js
+
+.PHONY: bundle
+bundle: dist/sanctuary.js dist/sanctuary.min.js
+
+.PHONY: dist/sanctuary.js
+dist/sanctuary.js:
+	mkdir -p -- '$(@D)'
+	cat \
+  node_modules/sanctuary-type-identifiers/index.js \
+  node_modules/sanctuary-type-classes/index.js \
+  node_modules/sanctuary-def/index.js \
+  index.js \
+  >dist/sanctuary.js
```

Comment:
If we don't need Browserify we should be able to avoid making `dist/sanctuary.js` a phony target:

```make
dist/sanctuary.js: \
		node_modules/sanctuary-type-identifiers/index.js \
		node_modules/sanctuary-type-classes/index.js \
		node_modules/sanctuary-def/index.js \
		index.js
	mkdir -p -- '$(@D)'
	cat -- $^ >'$@'
```

Reply:
Is this particular change resolved?

Reply:
I will push a commit to your branch. ;)